nodejs/examples/package.json

```diff
@@ -11,9 +11,22 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@lancedb/lancedb": "file:../",
-    "@xenova/transformers": "^2.17.2"
+    "@xenova/transformers": "^2.17.2",
+    "tsc": "^2.0.4"
```

Comment:
I think we should move `typescript` to `devDependencies`, but I don't think we should need `tsc`. Should be able to still run `npx tsc <file>`.
```suggestion
```